web/src/SidebarTriggerButton.tsx

```diff
@@ -46,6 +46,19 @@ export default class SidebarTriggerButton extends PureComponent<
 > {
   render() {
     let props = this.props
+    if (props.resourceName == "(Tiltfile)") {
```

Comment:
the protobuf has an isTiltfile property, can you use that instead?

Reply:
ooh that's much better, will try that, ty